packages/optimistic-oracle/test/index.js

```diff
@@ -5,39 +5,39 @@ const Main = require("../index.js");
 const winston = require("winston");
 const sinon = require("sinon");
 
+const hre = require("hardhat");
+const { runDefaultFixture, interfaceName } = require("@uma/common");
+const { getContract } = hre;
+const { assert } = require("chai");
+
 const { SpyTransport, spyLogLevel, spyLogIncludes } = require("@uma/financial-templates-lib");
-const { getTruffleContract } = require("@uma/core");
-const { addGlobalHardhatTestingAddress } = require("@uma/common");
 
-const OptimisticOracle = getTruffleContract("OptimisticOracle", web3);
-const MockOracle = getTruffleContract("MockOracleAncillary", web3);
-const Finder = getTruffleContract("Finder", web3);
-const Timer = getTruffleContract("Timer", web3);
+const Finder = getContract("Finder");
+const OptimisticOracle = getContract("OptimisticOracle");
+const MockOracle = getContract("MockOracleAncillary");
 
-contract("index.js", function () {
+describe("index.js", function () {
   let spy;
   let spyLogger;
 
   let pollingDelay = 0; // 0 polling delay creates a serverless bot that yields after one full execution.
   let errorRetries = 1;
   let errorRetriesTimeout = 0.1; // 100 milliseconds between performing retries
 
-  let finder;
-  let timer;
   let optimisticOracle;
   let mockOracle;
 
   before(async function () {
-    finder = await Finder.new();
-    timer = await Timer.new();
-    mockOracle = await MockOracle.new(finder.address, timer.address);
-
+    const accounts = await web3.eth.getAccounts();
+    await runDefaultFixture(hre);
     // Deploy a new OptimisticOracle.
-    optimisticOracle = await OptimisticOracle.new("120", finder.address, timer.address);
+    const finder = await Finder.deployed();
+    optimisticOracle = await OptimisticOracle.deployed();
```

Comment:
Had to change this to `.deployed` instead of `.new()` because otherwise the test on line fails. It looks like the bot sees the `OptimisticOracle` address as the one matching `deployments.get("OptimisticOracle")` which is doesn't seem to change even after I run `OptimisticOracle.new()`.

Instead, it just seems to use the address set when all the `deploy/` scripts are run in the test fixture.